renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:recommended"],
+  "dependencyDashboard": true,
+  "pinVersions": true,
+  "vulnerabilityAlerts": {
+    "groupName": "security updates",
+    "enabled": true,
+    "schedule": [],
+    "labels": ["security"],
+    "prCreation": "immediate"
+  },
+  "osvVulnerabilityAlerts": true,
```

Comment:
this is experimental, do we really want it? how do we track if it changes from experimental to stable or removed?

Reply:
I want to add an extra layer of security, but if we want to stay on the stable version, I will remove it.

https://docs.renovatebot.com/configuration-options/#osvvulnerabilityalerts